src/Query/QueryStringQuery.php

```diff
@@ -22,7 +22,9 @@ public function __construct(
         protected ?array $fields = null,
         ?float $boost = null,
         ?string $minimumShouldMatch = null,
-        ?string $rewrite = null
+        ?string $rewrite = null,
+        protected ?string $fuzziness = null,
```

Comment:
fuziness parameter is a good candidate for a `hasFuziness` trait. It can be enabled here and on MatchQuery, MultiMatchQuery

Reply:
Ok, I'll refactor it with a trait then